src/Components/Components/test/ParameterViewTest.Assignment.cs

```diff
@@ -5,9 +5,12 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography;
```

Comment:
🙄 VS

Reply:
ugh. I'm going to fix this have to rebuild, and only because its the crypto namespace :(